app/views/vacancies/_key_dates_sidebar.html.slim

```diff
@@ -7,9 +7,15 @@
       - if @vacancy.starts_on.present?
         li.key-dates-timeline__item
           h3.key-dates-timeline__title.govuk-heading-s
-            = t("jobs.starts_on")
+            = t("jobs.expected_start")
           p.key-dates-timeline__by
             = format_date(@vacancy.starts_on)
+      - elsif @vacancy.starts_asap.present?
```

Comment:
Why are you doing `@vacancy.starts_asap.present?` instead of `@vacancy.starts_asap?`?

Reply:
Mistake. I got rid of this at some point and then seem to have got rid of those changes. Will change now.